src/components/common.js

```diff
@@ -523,15 +523,27 @@ export const HeroWrapper = ({ showMenu = true, bigSubhead = false, children }) =
         backgroundRepeat: 'no-repeat', backgroundSize: '750px', backgroundPosition: 'right 0 top 0'
       }
     }, [
-      h1({ style: { fontSize: 54 } }, `Welcome to ${getAppName()}`),
+      Utils.cond(
+        [isTerra(), () => div({ style: { lineHeight: '2rem' } }, [
+          h1({ style: { fontSize: 54 } }, 'Welcome to'),
+          h1({ style: { fontSize: 54 } }, getAppName(true))
+        ])],
+        () => h1({ style: { fontSize: 54 } }, `Welcome to ${getAppName()}`)),
       div({ style: { margin: '1rem 0', width: 575, ...(bigSubhead ? { fontSize: 20, lineHeight: '28px' } : { fontSize: 16, lineHeight: 1.5 }) } }, [
-        `${getAppName(true)} is a ${Utils.cond(
+        `${getAppName(!isTerra())} is a ${Utils.cond(
```

Comment:
`getAppName`'s argument is whether you want the long name. I don't understand why we want to show short name for Terra but long name for others.

Reply:
Again was trying to match the wireframes which has the long name in the title (`Terra Community Workbench`) and the short name in the paragraph (`Terra is a…`). This is opposite to the other skins.

Reply:
Maybe ping UX again on whether they really want Terra to be the opposite of the others. If they do, that's fine.